setup.py

```diff
@@ -240,30 +240,42 @@ def get_extensions():
             extra_compile_args["nvcc"].append("-g")
             extra_link_args.append("/DEBUG")
 
+    this_dir = os.path.dirname(os.path.curdir)
+    extensions_dir = os.path.join(this_dir, "torchao", "csrc")
+    sources = list(glob.glob(os.path.join(extensions_dir, "**/*.cpp"), recursive=True))
+
+    extensions_cuda_dir = os.path.join(extensions_dir, "cuda")
+    cuda_sources = list(
+        glob.glob(os.path.join(extensions_cuda_dir, "**/*.cu"), recursive=True)
+    )
+
+    if use_cuda:
+        sources += cuda_sources
+
     use_cutlass = False
     if use_cuda and not IS_WINDOWS:
         use_cutlass = True
         cutlass_dir = os.path.join(third_party_path, "cutlass")
         cutlass_include_dir = os.path.join(cutlass_dir, "include")
+        cutlass_extensions_include_dir = os.path.join(cwd, extensions_cuda_dir)
     if use_cutlass:
         extra_compile_args["nvcc"].extend(
             [
                 "-DTORCHAO_USE_CUTLASS",
                 "-I" + cutlass_include_dir,
+                "-I" + cutlass_extensions_include_dir,
             ]
         )
-
-    this_dir = os.path.dirname(os.path.curdir)
-    extensions_dir = os.path.join(this_dir, "torchao", "csrc")
-    sources = list(glob.glob(os.path.join(extensions_dir, "**/*.cpp"), recursive=True))
-
-    extensions_cuda_dir = os.path.join(extensions_dir, "cuda")
-    cuda_sources = list(
-        glob.glob(os.path.join(extensions_cuda_dir, "**/*.cu"), recursive=True)
-    )
-
-    if use_cuda:
-        sources += cuda_sources
+    else:
+        # Remove CUTLASS-based kernels from the cuda_sources list.  An
+        # assumption is that these files will have "cutlass" in its
+        # name.
```

Comment:
Nit: can you update https://github.com/pytorch/ao/tree/main/torchao/csrc w/ the naming convention

Reply:
This addition was my suggestion.  Now, on second look, I don't think it was necessary, or even correct.  Namely, if CUTLASS not used, then because of `TORCHAO_USE_CUTLASS` not defined, the operators could still be compiled, and will just throw not implemented error.  But if this C++ code not compiled at all, I'm not sure if Python side of things will work properly (for example, when operators registered, etc.)  So I'd suggest we revert this change.

Reply:
I'm not sure what's the expected change. Can you elaborate? Thank you

@alexsamardzic Should be fine, since the ops are defined in Python (in `ops.py`). The C++ code only registers a CUDA impl for the op. Without it, the op just doesn't have a CUDA impl (i.e. no impl)

Reply:
Ah - indeed, I tested and when files with CUTLASS-based kernels are not included in the build, then the Python code will throw `NotImplementedError` itself.  OK then, let's keep it.

I believe @drisspg suggestion was just to add a note, at the end of `torchao/csrc/README.md` file, that all CUTLASS-based kernels should have `cutlass` in the name of its `.cu` files.